src/main/groovy/edu/oregonstate/mist/personsapi/mapper/JobsMapper.groovy

```diff
@@ -0,0 +1,34 @@
+package edu.oregonstate.mist.personsapi.mapper
+
+import edu.oregonstate.mist.core.JobObject
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+
+import java.sql.ResultSet
+import java.sql.SQLException
+
+public class JobsMapper implements ResultSetMapper<JobObject> {
+    public JobObject map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+
+        /*
+         * Hard code job statuses and those descriptions here since there is no validation table
+         */
+        def jobStatusDict = [
+            'A': 'Active',
+            'B': 'Leave without pay but with benefits',
+            'L': 'Leave without pay and benefits',
+            'F': 'Leave with full pay and benefits',
+            'P': 'Leave with partial pay and benefits',
+            'T': 'Terminated'
+        ]
+
+        new JobObject(
+            positionNumber: rs.getString('POSITION_NUMBER'),
+            fte: rs.getFloat('FTE'),
+            description: rs.getString('DESCRIPTION'),
+            beginDate: rs.getDate('BEGIN_DATE'),
+            endDate: rs.getDate('END_DATE'),
+            status: jobStatusDict[rs.getString('STATUS')]
```

Comment:
What if Banner adds a new status letter? Would this line throw an exception since the map wouldn't have the new status? I'm thinking for safety, we should return the literal status letter if the letter is not in `jobsStatusDict`. We should then have an integration test to make sure this field is more than just one character.